### DISCUSSION TITLE
Redwoodjs Integration

### DISCUSSION THREAD

**Discussion by masterbater** [ory/awesome-ory#40] [Ideas]
It would be nice if there is a way to integrate this to redwoodjs or just a guide. Default dbAuth used by redwoodjs also uses cookies but they also support more than 8 third party authentication https://redwoodjs.com/docs/authentication#third-party-providers-installation-and-setup
The only complete auth solution would be clerk and auth0, it would be nice if kratos will be added as an alternative open source complete auth solution and also integration with ory cloud.

All auth connectors has this hooks
 const { loading, isAuthenticated, logIn, logOut } = useAuth()

You have a sample integration to next.js and already used it to learn about ory kratos and use it to manually integrate to redwoodjs. But it would be great in the future if there are client sdk and admin sdk helpers for popular frameworks or libraries

**Comment by vinckr** [comment]
Hello @masterbater 
That is a great idea! We dont have an example for redwoodjs yet, but would love to add one to the ory/examples repository. 
Always open for contributions! If you (or someone else) are up to it feel free to reach out to me and I will give you a hand 🤗
